docs/admin/scale/index.md

```diff
@@ -0,0 +1,36 @@
+We regularly scale-test Coder against various reference architectures. Additionally, we provide a [scale testing utility](#scaletest-utility) which can be used in your own environment to give insight on how Coder scales with your deployment's specific templates, images, etc.
+
+## Reference Architectures
+
+| Environment                                       | Users         | Last tested  | Status   |
+| ------------------------------------------------- | ------------- | ------------ | -------- |
+| [Google Kubernetes Engine (GKE)](./gke.md)        | 50, 100, 1000 | Nov 29, 2022 | Complete |
```

Comment:
> Complete

It would be awesome if we can share more details here picturing the autoscaling behavior and the duration of tests.

BTW this data can be easily converted into a set of blog posts about loadtesting.

Reply:
> Complete

Also, some reference data about API latencies might be helpful also for us, so that we know if the Coder performance improved or decreased over time.

Reply:
Yeah, Test "Complete" is a very weak statement.  Like, we could totally fail the test and still say, well the test is _complete_.

Reply:
This was inspired by GitLab's [Performance and Stability](https://about.gitlab.com/handbook/engineering/quality/performance-and-scalability/) page. I wasn't sure the best way, in a table view, to show that we've validated a Coder deployment with `n` users, but agree that complete isn't the best term.

Perhaps `Validated`?

Reply:
The column could also be omitted and we put a ✅  or ⌛  next to each user count. I'm not exactly sure the best format for this information at the moment

Reply:
I presume that we could at least add a legend: green-yellow-red

_green_: everything went smooth 👍 SLAs (do we have any?) not impacted, platform performance not degraded
_yellow_: users can operate, but in a few cases we observed SLA being violated, for instance - due to high latency. We should describe specifically what went wrong
_red_: total disaster, platform misbehaves, not usable, etc.

In general, it would be awesome if we can automatically raise a Github issue for every performance test run and discuss the results there. BTW this is a good moment to "build" the SRE attitude in Coder :)

Reply:
The green yellow red system is a bit overkill for what we need right now. As we develop out our tests and automation we can start using it but we're nowhere near there yet. We also don't have any SLAs or criteria to base a yellow on yet.

Reply:
Sounds like an action item for me 👍